.github/workflows/test.yml

```diff
@@ -65,17 +65,10 @@ jobs:
           fetch-depth: 0 # 👈 Required to retrieve git history
       - name: Install dependencies
         run: yarn install --immutable
-      - name: Build Storybook 📚
-        run: yarn workspace @chanzuckerberg/eds-components run build:storybook
+      - name: Build Storybook and Publish to Chromatic
```

Comment:
Can you leave a comment here that we're using Chromatic to build Storybook because that's necessary to use `--only-changed`, and link to the Chromatic issue where they discussed that?

Reply:
Added comments as suggested, lmk if it's too verbose